converters/toZigbee.js

```diff
@@ -4952,6 +4952,195 @@ const converters = {
             return {state: {week: value}};
         },
     },
+    x5h_thermostat: {
+        key: ['system_mode', 'current_heating_setpoint', 'sensor', 'brightness_state', 'sound', 'frost_protection', 'week', 'factory_reset',
+            'local_temperature_calibration', 'protection_temp_limit', 'temp_diff', 'output_reverse', 'upper_temp', 'preset', 'child_lock'],
+        convertSet: async (entity, key, value, meta) => {
+            switch (key) {
+            case 'system_mode':
+                await tuya.sendDataPointBool(entity, tuya.dataPoints.x5hState, value === 'heat');
+                break;
```

Comment:
Wouldn't it be better to split this to separate converters? On the one hand, they will be reusable for other devices. On the other hand, there maybe no more devices with the same set of features on the same datapoints.

Reply:
This is also fine